packages/credential-ld/src/ld-credential-module.ts

```diff
@@ -35,7 +36,11 @@ export class LdCredentialModule {
     this.ldSuiteLoader = options.ldSuiteLoader
   }
 
-  getDocumentLoader(context: IAgentContext<IResolver>, attemptToFetchContexts: boolean = false) {
+  getDocumentLoader(
+    context: IAgentContext<IResolver>,
+    attemptToFetchContexts: boolean = false,
+    resolveFragments: boolean = false,
```

Comment:
setting this to true will unblock one of the errors in the 2020 suite, but will cause issues in all the others